lib/pure/math.nim

```diff
@@ -357,3 +391,36 @@ when isMainModule and not defined(JS):
   assert(lgamma(1.0) == 0.0) # ln(1.0) == 0.0
   assert(erf(6.0) > erf(5.0))
   assert(erfc(6.0) < erfc(5.0))
+when isMainModule:
+  # Function for approximate comparison of floats
+  proc floatIsEqual(x, y: float): bool = (abs(x-y) < 1e-9)
```

Comment:
I've searched through the code-base and can't find any relevant reference to `=~` or `==~`, so I assume you mean that I should implement it as a local proc instead of calling that proc `floatIsEqual`.  I've implemented that in 695b251; I used `==~` in case `=~` ever makes it into the standard library.